services/ui-src/src/components/forms/FormLayoutElements.tsx

```diff
@@ -3,6 +3,10 @@ import { Box, Heading, Text } from "@chakra-ui/react";
 export const SectionHeader = ({
   content,
   divider,
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Fixes `Warning: Invalid DOM property 'autosave'.` and `Warning: React does not recognize the 'validateOnRender' prop on a DOM element.`, but these aren't used anywhere in the component.